packages/bitcore-wallet-service/src/lib/pushnotificationsservice.ts

```diff
@@ -238,22 +295,35 @@ export class PushNotificationsService {
                   `The recipient list for this push notification is greater than the established limit (${Defaults.PUSH_NOTIFICATION_LIMIT})`
                 );
               }
-
               return next(err, notifications);
             },
             (notifications, next) => {
               async.each(
                 notifications,
-                (notification, next) => {
-                  this._makeRequest(notification, (err, response) => {
-                    if (err) logger.error('ERROR:' + err);
-                    if (response) {
-                      //                      logger.debug('Request status:  ' + response.statusCode);
-                      //                      logger.debug('Request message: ' + response.statusMessage);
-                      //                      logger.debug('Request body:  ' + response.request.body);
-                    }
-                    next();
-                  });
+                (notification: any, next) => {
+                  if (notification && notification.external_user_ids) {
+                    this._makeBrazeRequest(notification, (err, response) => {
+                      if (err) logger.error('ERROR:' + err);
+                      if (response) {
+                        //                      logger.debug('Request status:  ' + response.statusCode);
+                        //                      logger.debug('Request message: ' + response.statusMessage);
+                        //                      logger.debug('Request body:  ' + response.request.body);
+                      }
+                      next();
+                    });
+
+                    // if external_user_ids is not set use old version firebase token
+                  } else if (notification && notification.to) {
+                    this._makeRequest(notification, (err, response) => {
+                      if (err) logger.error('ERROR:' + err);
```

Comment:
I know this isn't originally your code, but cold we take this opportunity to make these error messages a little better?  This one and on line 306. Preferably some strings where we can uniquely identify where the error is originating instead of just `ERROR:`